app/assets/stylesheets/modules/landing.scss

```diff
@@ -0,0 +1,21 @@
+.welcome {
+  background-image: image-url("plane_wing.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  min-height: 600px;
+
+  h2 {
+    color: #ffa500;
```

Comment:
Color literals like `#ffa500` should only be used in variable declarations; they should be referred to via variable everywhere else.